src/sbvr-api/permissions.coffee

```diff
@@ -404,7 +505,27 @@ exports.setup = (app, sbvrUtils) ->
 	# A default api key middleware for convenience
 	exports.apiKeyMiddleware = apiKeyMiddleware = customApiKeyMiddleware()
 
-	exports.checkPermissions = checkPermissions = do ->
+	exports.checkPermissions = (req, actionList, resourceName, vocabulary) ->
+		getReqPermissions(req)
+		.then (permissionsLookup) ->
+			checkPermissions(permissionsLookup, actionList, vocabulary, resourceName)
```

Comment:
Isn't this now `undefined`?

Reply:
No, it's defined above where `_checkPermissions` was renamed to `checkPermissions`